client/actions/auth/signupActions.js

```diff
@@ -0,0 +1,14 @@
+import * as types from '../actionTypes';
```

Comment:
It is good practice to have your action files by resources (eg user) as opposed to actions (signin and signout actions). a `userActions` with signin and signout feels more appropriate.